spec/functional/func_table_screen_spec.rb

```diff
@@ -2,15 +2,11 @@
   tests PM::TestTableScreen
 
   def table_screen
-    @table_screen ||= begin
-      t = TestTableScreen.new(nav_bar: true)
-      t.on_load
-      t
-    end
+    @table_screen ||= TestTableScreen.new(nav_bar: true)
   end
 
   def controller
-    rotate_device to: :portrait, button: :bottom
+    # rotate_device to: :portrait, button: :bottom
```

Comment:
Commented this out, no problems on the tests, so I'm assuming it was unnecessary.